### ISSUE TITLE
Is there a '.gitmodules' file missing from this repo?

### ISSUE THREAD

**Issue by TristanTrim** [traverseda/inoperable#1]
trist@bizmuth:~/inoperable$ ./firstrun.sh 
can't load package: package github.com/oleiade/lane: no buildable Go source files in /home/trist/inoperable/src/github.com/oleiade/lane
trist@bizmuth:~/inoperable$ git submodule init 
fatal: no submodule mapping found in .gitmodules for path 'src/github.com/glycerine/rbtree'

**Comment by traverseda** [comment]
```
source env.sh
./firstrun.sh
```

?

**Comment by traverseda** [comment]
Huh, firstrun should be downloading those packages....

**Comment by traverseda** [comment]
Alright, so I got a chance to dig into this, after some caffeine. Remember I was complaining to you about "{src/,pkg/,bin}" being in the repo, and the golang ".gitignore" not removing it? Well the complaint stands. I'm still not sure why the golang gitignore doesn't ignore it.

Those repos were set up by the "go get" tool, go's package manager (well, except for bin/). Once I removed them from the repo, "go get" started working again.

Anyway, do a pull. Make sure those dirs don't exist, and rerun firstrun.sh.
